.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "OpenCorePkg"]
+	url = https://github.com/serdeliuk/OpenCorePkg.git
```

Comment:
Hi, are you sure to use your fork of OC instead of CloverHackyColor's?

Reply:
I have modified my OC indeed, https://github.com/serdeliuk/OpenCorePkg/commit/07db7299071b2391f4644fe81a35635b2fd70897
then requested another pull for OC

Reply:
Well spotted. I would have done the same mistake. He has to fork to create the pull request.
Once OpenCore is fixed and the PR merged, he can switch back our fork. Then, we merge PR for Clover itself.

Reply:
@serdeliuk OpenCorePKG PR merged. Could you now switch back to CloverHackyColor's fork for OpenCorePKG. Then commit that change.

Reply:
done

Reply:
This workflow build has a cache? Can be cleaned or completely deleted?

Reply:
You have to `git submodule update --remote` to pull the latest OC branch.

Reply:
Nice catch, thanks, done.

Reply:
Haha, nice, i didn't expected that an unused variable error actually to be a false positive and didn't checked more, if i try to compile the code with that variable commented-out i have the  same error now.
Could be something related to the building environment? I use  Mojave to build Clover.

Reply:
Is there any way to replicate the build env on my computer?

Reply:
So, there is something wrong with the testing env, the Privilege variable is set and used, i wonder why the unused error was in the first place. I have fixed back the OC and requested another pull

Reply:
It seems that the uploaded OpenCore.c has some conditionals missing, probably i applied a wrong patch, i am fixing the OC and request another pull

Reply:
I use Mojave too.
You can build with buildme script.

Reply:
Yes, i can, but the dev testing workflow in the beginning throwed an  error that i didn't had, probably will be better to understand how the workflow works and what env use, at least i can test locally prior uploading, this way i can save everyone's time :)
I found the issue, my last patch had some missing  compile time conditions , i will fix my OC and issue later today another pull request, then see

Reply:
> Try a build with "buildme" script.
Yes, this did the trick.
I wonder if anyone else  observed  that buildme script build a smaller efi than ebuild, is there any obvious reason why is doing this? 
```
-rw-r--r--   1 marc  admin   1.4M Sep 15 16:09 CLOVERX64.efi_buildme
-rw-r--r--   1 marc  admin   1.6M Sep 15 16:12 CLOVERX64.efi_ebuild

```

Reply:
buildme build clover 2 times, with different options. buildme always build DUET. So I wouldn't be surprised there is room for optimization...